### ISSUE TITLE
Mounting /system in Android

### ISSUE THREAD

**Issue by BruhDudeWew** [Gerriarob/arob#3]
Problem changed. How can I mount /system as read-write. I've tried mounting it in FX file explorer but no success. Then I tried mounting it with command but then it said:" mount: '/system' not in /proc/mounts"
How to solve that?

**Comment by Gerriarob** [comment]
did you checked that your android container is rooted? if your android is rooted than you dont need to mount or remount your system folder. i think you try to write from the chromeos shell to /system. this mountpoint does not exist. you need to install some terminal or root fileexplorer (mixplorer) inside android.

**Comment by BruhDudeWew** [comment]
> did you checked that your android container is rooted? if your android is rooted than you dont need to mount or remount your system folder. i think you try to write from the chromeos shell to /system. this mountpoint does not exist. you need to install some terminal or root fileexplorer (mixplorer) inside android.

I am using FX File Manager. But even if I try to mount it using FX File Manager, it won't work.

**Comment by Gerriarob** [comment]
You don’t need to mount, just open. Have you checked that you grant permission for root? Look in supersu if the file explorer have the permission if not, just grant root permission for fx file manager

**Comment by khanhtranngoccva** [comment]
Yes, the system partition is read-only. The script is missing these lines: 
echo "Substituting '|mount rootfs rootfs / remount bind rw' for '|mount rootfs rootfs / remount bind ro' in existing init.rc"
echo "A backup of init.rc will be stored as init.rc.old"

sed -i.old 's|mount rootfs rootfs / remount bind ro|mount rootfs rootfs / remount bind rw|g' $arc_system/../init.rc

If this line is not included, Chrome OS will mount the container's system partition read-only to increase security. Solve this by editing /usr/local/Android_Images/Mounted/init.rc manually right after the script finished by replacing "mount rootfs rootfs / remount bind rw" with "mount rootfs rootfs / remount bind rw", or you can wait for Gerri to fix it automatically.
